.github/workflows/e2e-ci-playwright.yml

```diff
@@ -0,0 +1,278 @@
+
+          # Validate JSON using jq
+          if jq empty playwright.env.json > /dev/null 2>&1; then
+            echo "playwright.env.json is valid JSON"
+          else
+            echo "ERROR: playwright.env.json is not valid JSON!"
+            exit 1
+          fi
+      - name: Execute tests
+        id: playwright-tests
+        env:
+          NODE_TLS_REJECT_UNAUTHORIZED: 0
+          BROWSER: ${{ github.event.inputs.browser || 'chrome' }}
+          WORKERS: ${{ github.event.inputs.workers || '4' }}
+          BASE_URL: https://prod.foo.redhat.com:1337/openshift/
+        run: |
+          LOGGING=quiet yarn start 2>&1 | grep --line-buffered -Ev '\[SERVER\] \[SERVE\]' &
+          NODE_TLS_REJECT_UNAUTHORIZED=0 yarn wait-on -i 5000 -t 120s https://prod.foo.redhat.com:1337/apps/openshift/fed-mods.json
+          NODE_TLS_REJECT_UNAUTHORIZED=0 yarn playwright-headless --grep "ci" --workers=$WORKERS
+      - name: Extract test results
+        id: test-results
+        if: always()
+        run: |
+          TEST_RESULTS_FILE="playwright-artifacts/results/test-results.json"
+
+          if [ -f "$TEST_RESULTS_FILE" ]; then
+            # Install jq if not available
+            if ! command -v jq &> /dev/null; then
+              sudo apt-get update && sudo apt-get install -y jq
+            fi
+            
+            # Extract test counts from JSON reporter (handle nested suites structure)
+            TOTAL=$(jq '[.. | .specs? // empty | .[]] | length' "$TEST_RESULTS_FILE")
+            PASSED=$(jq '[.. | .specs? // empty | .[] | .tests[]? | select(.status == "expected")] | length' "$TEST_RESULTS_FILE")
+            FAILED=$(jq '[.. | .specs? // empty | .[] | .tests[]? | select(.status == "unexpected")] | length' "$TEST_RESULTS_FILE")
+            SKIPPED=$(jq '[.. | .specs? // empty | .[] | .tests[]? | select(.status == "skipped")] | length' "$TEST_RESULTS_FILE")
+            
+            echo "total=${TOTAL}" >> $GITHUB_OUTPUT
+            echo "passed=${PASSED}" >> $GITHUB_OUTPUT
+            echo "failed=${FAILED}" >> $GITHUB_OUTPUT
+            echo "skipped=${SKIPPED}" >> $GITHUB_OUTPUT
+          else
+            echo "total=0" >> $GITHUB_OUTPUT
+            echo "passed=0" >> $GITHUB_OUTPUT
+            echo "failed=0" >> $GITHUB_OUTPUT
+            echo "skipped=0" >> $GITHUB_OUTPUT
+          fi
+      - uses: actions/upload-artifact@v4
+        if: ${{ failure() }}
+        with:
+          name: playwright-e2e-facts
+          retention-days: 5
+          path: |
+            playwright-artifacts/
+      - name: Get current timestamp
+        id: timestamp
+        if: always()
+        run: echo "now=$(date +%s)" >> $GITHUB_OUTPUT
+      - name: Publish results to Slack
+        if: always()
+        uses: slackapi/slack-github-action@v2.1.1
+        with:
+          webhook: ${{ secrets.SLACK_PLAYWRIGHT_TESTS_PUBLISHER_WEBHOOK_URL }}
```

Comment:
Why do you want to send reports to slack every time? This will run inside every PR. Do we want to get a report for every single run in every single PR? 🤔

Reply:
Currently, the workflow runs for every PR and is marked as continue-on-error: true during the cooling period. I would like to analyze the test results from a single, centralized location rather than reviewing each PR individually. Once everything is stable and we are ready to officially replace the existing Cypress CI, we can remove the continue-on-error: true setting and the Slack notification configuration.
By the way, Slack notifications are planned to be introduced as part of the Smoke CI, which will be added to the system soon.